src/models/jira-client.test.ts

```diff
@@ -6,13 +6,13 @@ describe("JiraClient", () => {
 	describe("isAuthorized()", () => {
 		let jiraClient: any;
 
-		beforeEach(() => {
+		beforeEach(async () => {
 			const installation: any = {
 				jiraHost,
 				sharedSecret: "secret"
 			};
 
-			jiraClient = new JiraClient(installation, logger);
+			jiraClient = await JiraClient.getNewClient(installation, logger);
```

Comment:
Why this pattern?

Reply:
I couldn't find a way to make constructor async (to decrypt)

Reply:
ah right. I guess that's acceptable for now until we don't use the FF anymore.  In the future, I think we'll just send the whole installation object and we'd only decrypt the secret when a request is made.